src/js/post-911-gib-status/components/UserInfoSection.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { getDateFormatted, getPercentFormatted } from '../utils/helpers';
+
+class UserInfoSection extends React.Component {
+  render() {
+    const { userData } = this.props;
+
+    // Used to get today's date to show information current of
+    const today = new Date();
+    const todayFormatted = getDateFormatted(today);
+
+    // Check if percent is a number
+    let percentageBenefit = 'unavailable';
+    if (getPercentFormatted(userData.percentageBenefit)) {
+      percentageBenefit = getPercentFormatted(userData.percentageBenefit);
+    }
+
+    return (
+      <div className="gibstatus">
+        <h3 className="section-header">{userData.firstName} {userData.lastName}</h3>
```

Comment:
I feel like they should be as consistent as possible, unless there's a good reason they're not?